src/pages/ApplicationControls.jsx

```diff
@@ -0,0 +1,143 @@
+    if (eventData) {
+      setEventDate(eventData.startAt);
+      setappId(eventData.id);
+    }
+  }, [eventData]);
+
+  const handleSubmit = async () => {
+    if (startDate && endDate) {
+      setAlert({
+        type: 'success',
+        message: 'The dates have been submitted successfully.',
+        isVisible: true,
+      });
+
+      try {
+        await control({
+          start_at: new Date(startDate).toISOString(),
+          end_at: new Date(endDate).toISOString(),
+        });
+      } catch (error) {
+        setAlert({
+          type: 'error',
+          message: 'Please fill in all dates before submitting.',
+          isVisible: true,
+        });
+      }
+    } else {
+      setAlert({
+        type: 'error',
+        message: 'Please fill in all dates before submitting.',
+        isVisible: true,
+      });
+    }
+  };
+
+  const handleCloseAlert = () => {
+    setAlert({ ...alert, isVisible: false });
+  };
+
+  return (
+    <Flex
+      className="application-controls"
+      p={4}
+      width="100%"
+      maxWidth="1200px"
+      direction={{ base: 'column', md: 'row' }}
+      gap={4}
+    >
+      <Box flex="1" borderWidth="1px" borderRadius="lg" boxShadow="md" p={4} mr={6}>
+        <Heading as="h1" size="lg" mb={6}>
+          Application Controls
+        </Heading>
+        <Flex direction="column" gap={4}>
+          <FormControl id="start-date" mt={4}>
+            <FormLabel fontSize="md" fontWeight="bold">
+              Start Date
+            </FormLabel>
+            <Input
+              type="date"
+              name="start-date"
+              value={startDate}
+              onChange={(e) => setStartDate(e.target.value)}
+            />
+          </FormControl>
+          <FormControl id="end-date" mt={4}>
+            <FormLabel fontSize="md" fontWeight="bold">
+              End Date
+            </FormLabel>
+            <Input
+              type="date"
+              name="end-date"
+              value={endDate}
+              onChange={(e) => setEndDate(e.target.value)}
+            />
+          </FormControl>
+          <Button colorScheme="green" mt={4} onClick={handleSubmit}>
+            Submit
+          </Button>
+          {alert.isVisible && (
+            <Alert status={alert.type} mt={4} borderRadius="md">
+              <AlertIcon />
+              <Box flex="1">
+                <AlertTitle>{alert.type === 'success' ? 'Success!' : 'Error'}</AlertTitle>
+                <AlertDescription>{alert.message}</AlertDescription>
+              </Box>
+              <CloseButton onClick={handleCloseAlert} position="absolute" right="8px" top="8px" />
+            </Alert>
+          )}
+        </Flex>
+      </Box>
+      <Box
+        flex="1"
+        borderWidth="1px"
+        borderRadius="lg"
+        boxShadow="md"
+        p={4}
+        display="flex"
+        justifyContent="center"
+        alignItems="center"
+      >
+        <CountdownTimer date={new Date(eventDate)} />
```

Comment:
I feel like this countDown timer should be counting down to the time the applications open/close, not when the event actually starts. because the start_at and end_at fields in the form table represent when the applications open/close.

the start_at column in the forms table is also not when the event starts, it's when the applications open

if currentTime < startDate:
countDown to opening of applications

if currentTime > startDate && currentTime < endDate
countDown to closing of applications

else // applications are closed
display some text that says applications are closed